js/models.js

```diff
@@ -104,6 +104,31 @@ class StoryList {
       /*
     old code, might be useful at some point */
   }
+
+  /* Recieves a user obj and a storyId
+  *   -verifies story belongs to user
+  *   -deletes story from api
+  */
+  async deleteStoryFromList(user, storyId){
+    const belongsToUser = user.ownStories.some((story) => (story.storyId == storyId));
+    if(belongsToUser){
+      try{
+        await axios({
+          url: `${BASE_URL}/stories/${storyId}`,
+          method: "DELETE",
+          data: {
+            "token": user.loginToken
+          }
+        });
+        currentUser = await User.updateUser();
+        displayMyStories();
```

Comment:
`displayMyStories()`: models should never call functions from outside. Their responsibility is only to fetch/send data from/to the persistence layer and return the result of each operation to whomever called it (either data or error/success status). Please refactor to remove this call.